apps/api-extractor/src/collector/CollectorEntity.ts

```diff
@@ -59,13 +47,72 @@ export class CollectorEntity {
     this._sortKey = undefined; // invalidate the cached value
   }
 
+  /**
+   * If this symbol is exported from the entry point, the list of export names.
+   *
+   * @remarks
+   * Note that a given symbol may be exported more than once:
+   * ```
+   * class X { }
+   * export { X }
+   * export { X as Y }
+   * ```
+   */
+  public get exportNames(): ReadonlySet<string> {
+    if (!this._exportNamesSorted) {
+      Sort.sortSet(this._exportNames);
+      this._exportNamesSorted = true;
+    }
+    return this._exportNames;
+  }
+
+  /**
+   * If exportNames contains only one string, then singleExportName is that string.
+   * In all other cases, it is undefined.
+   */
+  public get singleExportName(): string | undefined {
+   return this._singleExportName;
+  }
+
+  /**
+   * This is true if exportNames contains only one string, and the declaration can be exported using the inline syntax
+   * such as "export class X { }" instead of "export { X }".
+   */
+  public get emitWithExportKeyword(): boolean {
```

Comment:
>emitWithExportKeyword [](start = 13, length = 21)

Maybe `shouldInlineExport`

Reply:
Good suggestion

---
In reply to: [248127101](https://github.com/Microsoft/web-build-tools/pull/1002#discussion_r248127101) [](ancestors = 248127101)